### PR TITLE
analytics - datadirization

### PR DESCRIPTION
**Pull request by pmauduit**
This datadirization is a bit more complex than the others, because I had to wire spring JSP subsystem instead of the old one, so that I was able to use the same techniques to get the variables from the datadir.

* [x] compilation
* [x] testsuite (no tests :()
* [x] runtime tests with `georchestra.datadir`
* [x] runtime tests without `georchestra.datadir`

### CODE REVIEW

File: analytics/pom.xml [hunk 1]
```diff
@@ -170,4 +186,135 @@
...
+          <plugin>
+            <groupId>net.sf.debian-maven</groupId>
+            <artifactId>debian-maven-plugin</artifactId>
+            <version>1.0.6</version>
+            <configuration>
+              <packageName>georchestra-analytics</packageName>
+              <packageDescription>Debian package for geOrchestra's Analytics Webapp.</packageDescription>
+              <projectOrganization>geOrchestra</projectOrganization>
+              <maintainerName>Pierre Mauduit</maintainerName>
+              <maintainerEmail>pierre.mauduit@gmail.com</maintainerEmail>
```
Comment (fvanderbiest) [inline analytics/pom.xml]: PSC

File: analytics/pom.xml [hunk 2]
```diff
@@ -170,4 +186,135 @@
...
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>net.sf.debian-maven</groupId>
+            <artifactId>debian-maven-plugin</artifactId>
+            <version>1.0.6</version>
+            <configuration>
+              <packageName>georchestra-analytics</packageName>
+              <packageDescription>Debian package for geOrchestra's Analytics Webapp.</packageDescription>
```
Comment (fvanderbiest) [inline analytics/pom.xml]: name

File: analytics/src/deb/resources/etc/georchestra/analytics/analytics.properties [hunk 3]
```diff
@@ -0,0 +1,4 @@
+dlJdbcUrlOGC=jdbc:postgresql://localhost:5432/georchestra?user=www-data&password=www-data
+dlJdbcUrlDLForm=jdbc:postgresql://localhost:5432/georchestra?user=www-data&password=www-data
```
Comment (fvanderbiest) [inline analytics/src/deb/resources/etc/georchestra/analytics/analytics.properties]: missing comments regarding the role of these vras

File: analytics/pom.xml (hunk 1 above)
Comment (pmauduit) [inline analytics/pom.xml]: done

File: analytics/src/deb/resources/etc/georchestra/analytics/analytics.properties (hunk 3 above)
Comment (pmauduit) [inline analytics/src/deb/resources/etc/georchestra/analytics/analytics.properties]: done (this is currently also undocumented in master)

File: analytics/pom.xml (hunk 2 above)
Comment (pmauduit) [inline analytics/pom.xml]: changed the packageDescription (not sure it was what you meant though)

**Pull request merged**